### PR TITLE
Data fix

### PR DESCRIPTION
**Pull request by Parsifal-2045**
This pull request was created to update the data folder. It now contains the necessary setup data, some input samples and reference toyDetectors outputs for validation. 
The Makefile has been updated to be more in line with Pixeltrack standalone, with the new data dependencies, a more recent version of Alpaka and a cvmfs source for SYCL.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -549,10 +552,11 @@ external_eigen: $(EIGEN_BASE)
 
 $(EIGEN_BASE):
 	# from Eigen master branch as of 2021.08.18
-	git clone -b cms/master/82dd3710dac619448f50331c1d6a35da673f764a https://github.com/cms-externals/eigen-git-mirror.git $@
+	#git clone -b cms/master/82dd3710dac619448f50331c1d6a35da673f764a https://github.com/cms-externals/eigen-git-mirror.git $@
```
Comment (felicepantaleo) [inline Makefile:555]: Do we need Eigen?

File: data/input/url_input.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+https://cernbox.cern.ch/index.php/s/7YAFBhDIJMLIFBE/download
```
Comment (felicepantaleo) [inline data/input/url_input.txt:1]: Let's update the original download package instead of adding a new one

File: data/url.txt [hunk 3]
```diff
@@ -1 +1 @@
-https://fpantale.web.cern.ch/fpantale/heterogeneous_clue/heterogeneous_clue_input_data.tar.gz
\ No newline at end of file
+https://mkortela.web.cern.ch/mkortela/patatrack/standalone/data_v2.tar.gz
```
Comment (felicepantaleo) [inline data/url.txt:1]: why do we need pixeltracking data?

File: data/output/reference/url_reference.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+https://cernbox.cern.ch/index.php/s/ggOk369J4R2Rw1y/download
```
Comment (felicepantaleo) [inline data/output/reference/url_reference.txt:1]: let's put also the reference in the same archive as the input as they are connected.

**Review by felicepantaleo — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Parsifal-2045) [reply Makefile:555]: We don't. Removed from the Makefile

File: data/url.txt (hunk 3 above)
Comment (Parsifal-2045) [reply data/url.txt:1]: Currently investigating data usage in EventProcessor and Source

Comment (Parsifal-2045) [reply data/url.txt]: Issue solved, branch has been updated

**Pull request merged**